### PR TITLE
[NEW] Add option to enable X-Frame-options header to avoid loading inside any Iframe

### PR DESCRIPTION
**Pull request by MarcosSpessatto**
Closes https://github.com/RocketChat/Rocket.Chat/issues/843
Closes https://github.com/RocketChat/Rocket.Chat/issues/14669

### CODE REVIEW

File: server/startup/migrations/v148.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { Migrations } from '../../../app/migrations';
+import { Settings } from '../../../app/models';
+
+Migrations.add({
+	version: 148,
```
Comment (rodrigok) [inline server/startup/migrations/v148.js]: Please import the migration in `server/startup/migrations/index.js`.

File: server/startup/migrations/v148.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+	up() {
+		Settings.upsert({ _id: 'Allow_Loading_In_Iframe' }, {
```
Comment (rodrigok) [inline server/startup/migrations/v148.js]: Add a comment here:
```
Enable iframe usage for existant RC installations.
```

File: app/cors/server/cors.js [hunk 3]
```diff
@@ -48,6 +48,9 @@ WebApp.rawConnectHandlers.use(function(req, res, next) {
 	if (/^\/(api|_timesync|sockjs|tap-i18n)(\/|$)/.test(req.url)) {
 		res.setHeader('Access-Control-Allow-Origin', '*');
 	}
+	if (!settings.get('Allow_Loading_In_Iframe')) {
```
Comment (rodrigok) [inline app/cors/server/cors.js]: This is a bit confusing, I'd suggest to change `Allow_Loading_In_Iframe` to `Iframe_Restrict_Access`, and when enabled (the default behavior) set the `X-Frame-Options`. In this case I would change the setting `X_Frame_Options` to `Iframe_X_Frame_Options `

**Comment by sampaiodiego** [comment]
Can you fix the conflicts please?

**Comment by Dipmalya007** [comment]
Congratulations on getting this fixed.

**Pull request merged**